docs/streams/developer-guide.html

```diff
@@ -1554,13 +1557,13 @@
                     // Reducing a KGroupedStream
                     KTable&lt;String, Long&gt; aggregatedStream = groupedStream.reduce(
                         (aggValue, newValue) -> aggValue + newValue, /* adder */
-                        "reduced-stream-store" /* state store name */);
+                    );
 
                     // Reducing a KGroupedTable
                     KTable&lt;String, Long&gt; aggregatedTable = groupedTable.reduce(
                         (aggValue, newValue) -> aggValue + newValue, /* adder */
                         (aggValue, oldValue) -> aggValue - oldValue, /* subtractor */
-                        "reduced-table-store" /* state store name */);
+                    );
```

Comment:
You must remove the comma `,` in the line before, now that we removed the last parameter.